woocommerce/woocommerce-ios
# [TTPUK] Use country code in card configuration

Part of: #10890
Merge after: #10909


## Description


We'll add Tap to Pay on iPhone support for UK WooPayments merchants. 

This PR uses the previously extracted CountryCode enum in the CardPresentPaymentsConfiguration (as well as in Shipping Labels) instead of a raw string, for safer country checks and simpler localization in the app layer. This will enable us to keep app layer checks out of the business logic layer configuration.

## Testing instructions


This PR doesn't make any functional changes. Affected areas are CardPresentPaymentConfiguration, and Shipping Labels.

In particular, check for: 
- Country code should still be present in Card Present Payment related analytic events
- Shipping Labels should be promoted on the Order Details page for completed orders in the US – either as a prompt to install the plugin, or a button to purchase a shipping label if it's already installed.
- The `From` address should correctly show the country (as a code initially, then with its name when you tap on `Next` and see the detailed form)

It's worth checking some payment flows – the TTPUK work is not in this tree, so on this PR, only US should offer TTP and card readers, while GB and CA should offer card readers only.

---
